__tests__/munge.test.js

```diff
@@ -0,0 +1,205 @@
+require('dotenv').config();
+
+
+const { mungeLocationResponse } = require('../lib/munge');
+
+describe('app routes', () => {
+  describe('munge functions', () => {
+
+    test('munges location data and turns into a better object', async () => {
```

Comment:
glad you caught to test the munge function, and not the route which is what you previously had.   Full points for location!